names/module.py

```diff
@@ -0,0 +1,250 @@
+from typing import Optional
+
+import discord
+from discord.ext import commands
+
+from pie import check, utils, i18n, logger
+from modules.base.admin.database import BaseAdminModule
+from modules.boards.karma.database import KarmaMember
+
+from .database import Price
+
+_ = i18n.Translator("modules/fun").translate
+bot_log = logger.Bot.logger()
+
+
+class Names(commands.Cog):
+    """Pay by karma and I change your name."""
+
+    def __init__(self, bot: commands.Bot):
+        # Check if dependency module is loaded.
+        if not BaseAdminModule.get("boards.karma").enabled:
+            raise Exception(
+                "Unable to load names module due to missing module `boards.karma`."
+            )
+
+        Price.add_commnand("set", 500)
+        Price.add_commnand("unset", 200)
+        self.bot = bot
+
+    @commands.guild_only()
+    @check.acl2(check.ACLevel.MEMBER)
+    @commands.cooldown(rate=5, per=20.0, type=commands.BucketType.user)
+    @commands.command()
+    async def prices(self, ctx):
```

Comment:
There may be problems with namespace clashes with other commands. I think that we should place it all under `nickname` command group:
```
nickname
  prices
  set <new nickname>
  set-prices <set> <reset>
```

Reply:
I think `shop` might be better, since `nickname` seems to vague for me.

Reply:
When using name shop, it's mean that more thing it's sold line name, color, some special role etc.

In the end the command should be `nickname` with alias `name` because this command only working with name. As a feature when there will be more items in shop the nickname command can go under the shop command.

But as issue #3 states the shop new name should be `names`